scripts/packages/VSCodeServer/src/repl_protocol.jl

```diff
@@ -0,0 +1,35 @@
+JSONRPC.@dict_readable struct ReplGetVariablesRequestReturn <: JSONRPC.Outbound
+    type::String
+    value::String
+    name::String
+    id::Int
+    haschildren::Bool
+    lazy::Bool
+    icon::String
+    canshow::Bool
+end
+
+JSONRPC.@dict_readable struct ReplRunCodeRequestParams <: JSONRPC.Outbound
+    filename::String
+    line::Int
+    column::Int
+    code::String
+    mod::String
+    showCodeInREPL::Bool
+    showResultInREPL::Bool
+end
+
+JSONRPC.@dict_readable struct ReplRunCodeRequestReturn <: JSONRPC.Outbound
+    inline::String
+    all::String
+    iserr::Bool
+end
+
+const repl_runcode_request_type = JSONRPC.RequestType("repl/runcode", ReplRunCodeRequestParams, ReplRunCodeRequestReturn)
+const repl_getvariables_request_type = JSONRPC.RequestType("repl/getvariables", Nothing, Vector{ReplGetVariablesRequestReturn})
+# TODO Return type
+const repl_getlazy_request_type = JSONRPC.RequestType("repl/getlazy", Int, Vector{Any})
```

Comment:
@pfitzseb could you try to create the types that we need for the return value here? I didn't fully understand the structure there, and was also not sure about good names. Essentially right now there are some dicts returned, and ideally we would create a type for those instead. Could you maybe open a PR with that against this branch?

Reply:
Actually, thinking about this a bit more: why don't we merge this PR here (it works), and then you can open a new PR against `master` at some point that uses static types for everything?

Reply:
Yeah, let's do that.